### ISSUE TITLE
Unimplemented zone record type= TYPE65534

### ISSUE THREAD

**Issue by hmoffatt** [StackExchange/dnscontrol#2321]
I'm trying to import a DNSSEC-signed zone from a BIND server 9.16 server using AXFR, and getting

```
failed GetZone gzr: rrToRecord: Unimplemented zone record type= (mydomain.com.	0	CLASS1	TYPE65534	\# 5 0d64690001)
```

`dig`  also shows these records. Seems to be related to NSEC somehow:

```
$ dig axfr @localhost mydomain.com | grep TYPE
mydomain.com.	0	IN	TYPE65534 \# 5 0D64690001
mydomain.com.	0	IN	TYPE65534 \# 5 0DD8A40001
mydomain.com.	0	IN	RRSIG	TYPE65534 13 2 0 20230516072347 20230416070048 55460 mydomain.com. [stuff...]
mydomain.com.	600	IN	NSEC	_02c9d8a419f65e1d83567f40a5daf3e0.mydomain.com. A NS SOA MX TXT AAAA RRSIG NSEC DNSKEY TYPE65534
```

Seems to be, err, normal? It's in the example in the BIND docs, though not specifically described: https://kb.isc.org/docs/aa-00711

Should dnscontrol just be ignoring these records?

**Comment by tlimoncelli** [comment]
DNSControl's support of DNSSEC is pretty minimal.  It can enable automatic DNSSEC generation at the provider but not much else.  It definitely doesn't support the NSEC or RRSIG records.

I confess that I don't know much about DNSSEC.

Can you get the zone file some other way, delete those records, and do the import from the file?

**Comment by hmoffatt** [comment]
I was able to use the BIND zone file directly, so I have a workaround. Still, it might be useful if the AXFRDDNS provider could just ignore these records.

**Comment by tlimoncelli** [comment]
CC  @hnrgrgr (owner of the AXFRDDNS provider)

**Comment by hnrgrgr** [comment]
Look like `TYPE65534` is only a internal record in which BIND store an intermediate state while signing zone with NSEC3. And it should not be exported in an AXFR requests, IIUC.

  https://bind9-users.isc.narkive.com/zX29ay0j/rndc-signing-list-not-working#post2

The field does not to appear be specified in an RFC and thus is not handled by the `dns` library.

I added a proposed fix in #2335: where such records are "ignored" like any other DNSSEC records. But I have not been able to reproduce the failure and I did not test the proposed fix.
